dom.js

```diff
@@ -0,0 +1,61 @@
+const SIZE = 10;
+
+/*
+    1. Multiplying board
+*/
+function createMultiplyingBoard() {
+    let table = document.getElementById("multiplication");
+    let tableData = document.createDocumentFragment();
+    
+    createHeader(tableData,table);
+    
+    for (let i=1; i < SIZE; i ++) {
+        let headerLine = document.createElement("tr");
+        let lineNumber = document.createElement("th");
+        lineNumber.textContent = i;
+        headerLine.appendChild(lineNumber);
+
+        for (let j=1; j < SIZE; j ++) {
+            let cell = document.createElement("td");
+            cell.textContent = i*j;
+            if (i * j % 3 === 0) {
```

Comment:
In general, one-letter variables, and all kind of abbreviation variable names are a bad practice. It is standard to use `i` but since you already have a loop inside of a loop, I'd use more descriptive names, something like `rowValues` and `cellValue`